src/onnx/parse_resize.cpp

```diff
@@ -71,6 +71,96 @@ const auto& get_original_idx_op(const std::string& mode)
     return idx_ops.at(mode);
 }
 
+using vvv = std::vector<std::vector<std::vector<std::size_t>>>;
```

Comment:
onnx runtime uses a naming scheme like v, v1, v2. Do we want to do the same to make it easier to read?

Reply:
This is only used in one place so can we just remove the typedef?

Reply:
removed the typedef